en/publish/publish-qqplay.md

```diff
@@ -1,67 +1,66 @@
+- Support remote __QQ__ resource loading, caching, and version control of cache resources for __QQ Play__ platform standard.
 
-构建完成后访问项目文件夹，在 build 目录下就会生成一个 qqplay 的发布包：
+## Use __Cocos Creator__ to launch a mobile phone with one click __QQ Play__
 
-![](./publish-qqplay/package.jpeg)
+**Ready to work:**
 
-2、将 qqplay 文件夹中的内容拷贝到 Bricks 引擎工程的 Res 文件夹中：
+Obtain the Bricks engine project (this project is temporarily unavailable, developers can apply to the __Cocos Creator__ engine team for internal testing, or directly apply to Tencent for bitbucket library read access)
 
-![](./publish-qqplay/paste.jpeg)
+**Publishing process:**
 
-3、打开 Bricks 引擎中的 Xcode 工程，就可以直接进行玩一玩的编译调试了。具体可查看 [Bricks 引擎官方调试文档](http://hudong.qq.com/docs/engine/introduce/safari_debug.html)。
+1. Use __Cocos Creator__ to open the project project you want to publish, and select __QQ Play__ platform in the __Build...__ panel. If necessary, you can set the `Remote server address` option. Then click on **Build**.
 
-与这篇文档搭配的还有一段教学视频，可以结合视频中的操作来理解，更加直观。
+![](./publish-qqplay/build.jpeg)
 
-{% raw %}
+After the build is complete, a __qqplay__ release package will be generated in the __build__ directory:
 
-<iframe frameborder="0" width="100%" height="440" src="https://v.qq.com/iframe/player.html?vid=m055288q7cl&tiny=0&auto=0" allowfullscreen></iframe>
+![](./publish-qqplay/package.jpeg)
 
-{% endraw %}
+2. Copy the contents of the __qqplay__ folder to the __Res__ folder of the Bricks Engine project:
 
-<br>
+![](./publish-qqplay/paste.jpeg)
+
+3. Open the Xcode project in the Bricks engine, you can directly play and compile and debug. For details, see the [Bricks Engine Official Debugging Document](http://hudong.qq.com/docs/engine/introduce/safari_debug.html).
 
-## QQ「玩一玩」已知问题：
+## __QQ Play__ Known issues:
 
-目前暂时仅支持在 Mac 上使用 Xcode 进行编译开发。<br>
-此外，我们对 QQ「玩一玩」的适配工作还未完全结束，目前仍不支持以下模块：
+Currently, only Xcode is used for compiling and developing on Mac. In addition, our adaptation of __QQ Play__ has not been completely completed, and the following modules are still not supported:
 
-- Label：不支持 TTF，系统字体也只能使用默认字体
-- 粒子系统：不支持 plist 中存储贴图数据，只支持贴图 URL
+- Label: TTF is not supported, and system fonts can only use default fonts.
+- Particle system: Does not support storing texture data in plist, only supports texture URL
 - WebView
 - VideoPlayer
 - DeviceMotion
 
-以上功能预计在以后的更新中会逐步支持，我们也将持续与 QQ「玩一玩」工程师们紧密沟通，不断优化适配效果。
+The above functions are expected to be gradually supported in future updates, and we will continue to communicate closely with __QQ Play__ engineers to continuously optimize the adaptation effect.
 
-## 接入注意事项：
+## Access Notes:
 
-- 首包必须在 10M 以内，后续的资源可以部署在腾讯 CDN 上用 https 进行加载。
-- 上线前需要软件著作权，并提供版号申请证明，回执或者受理截图。
-- 除腾讯工作室外开发的游戏外，名称不允许出现 QQ、腾讯、天天、全民、欢乐等官方字眼。
-- 游戏必须有自己的大厅，大厅需要具备开始游戏（含邀请好友）、道具商城、活动入口。
-- 游戏内需要具备分享至 QQ 好友 / QQ 空间的能力。
-- 游戏需要使用自建后台，必须接 openkey 鉴权。
+- The first package must be within 10MB, and subsequent resources can be deployed on the Tencent CDN with https.
```

Comment:
"subsequent resources can be deployed on the Tencent CDN using https for loading" maybe be better?

Reply:
I see, yes.